deployments/main.tf

```diff
@@ -191,39 +191,46 @@ resource "google_storage_notification" "attestation-notification" {
 
 ## Container resources
 
-resource "google_container_registry" "registry" {
-  project  = var.project
-  location = "US"
+resource "google_artifact_registry_repository" "registry" {
+  provider = google-beta
+  location = "us-central1"
+  repository_id = "oss-rebuild-repo"
```

Comment:
This should probably not be hard-coded.

Reply:
How about...
```suggestion
  repository_id = "oss-rebuild-service-images"
```
?

Reply:
ah what i was saying that including "oss-rebuild" as a prefix would make the URI: "pkg.dev/oss-rebuild/oss-rebuild...". So something like "service-images" or "services" would be preferable.

Reply:
Ah, makes sense.
```suggestion
  repository_id = "service-images"
```